nifi-nar-bundles/nifi-cybersecurity-bundle/nifi-cybersecurity-processors/src/main/java/org/apache/nifi/processors/cybersecurity/CompareFuzzyHash.java

```diff
@@ -110,24 +107,25 @@
     public static final PropertyDescriptor MATCHING_MODE = new PropertyDescriptor.Builder()
             .name("MATCHING_MODE")
```

Comment:
Sorry, I missed this one earlier. I think the style elsewhere is skinny snake case - `matching-mode`.